src/pages/index.js

```diff
@@ -123,7 +123,7 @@ function handleProfileFormSubmit(inputValues) {
       console.log(err);
     })
     .finally(() => {
-      profileEditPopup.renderLoading(false, "Save");
+      profileEditPopup.renderLoading(false, "Loading...");
```

Comment:
Needs correction: You should start the loading before api request and end it here

Reply:
i do not understand what this means? would you explain it to me pretty please?

Reply:
You've fixed that on line 114. Great!